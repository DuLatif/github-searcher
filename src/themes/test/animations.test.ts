import { describe, expect, test } from "vitest";
import { readFileSync } from "fs";

describe("Theme Animations", () => {
  test("should match scss animations snapshot", () => {
    expect(readFileSync(__dirname + "./../animations.scss")).toMatchSnapshot();
  });
});
