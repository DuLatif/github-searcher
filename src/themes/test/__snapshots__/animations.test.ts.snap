// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Theme Animations > should match scss animations snapshot 1`] = `
{
  "data": [
    36,
    115,
    107,
    101,
    108,
    101,
    116,
    111,
    110,
    58,
    32,
    115,
    107,
    101,
    108,
    101,
    116,
    111,
    110,
    32,
    49,
    115,
    32,
    105,
    110,
    102,
    105,
    110,
    105,
    116,
    101,
    32,
    97,
    108,
    116,
    101,
    114,
    110,
    97,
    116,
    101,
    59,
    13,
    10,
    13,
    10,
    64,
    107,
    101,
    121,
    102,
    114,
    97,
    109,
    101,
    115,
    32,
    115,
    107,
    101,
    108,
    101,
    116,
    111,
    110,
    32,
    123,
    13,
    10,
    32,
    32,
    48,
    37,
    32,
    123,
    13,
    10,
    32,
    32,
    32,
    32,
    98,
    97,
    99,
    107,
    103,
    114,
    111,
    117,
    110,
    100,
    45,
    99,
    111,
    108,
    111,
    114,
    58,
    32,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    50,
    53,
    59,
    13,
    10,
    32,
    32,
    125,
    13,
    10,
    32,
    32,
    49,
    48,
    48,
    37,
    32,
    123,
    13,
    10,
    32,
    32,
    32,
    32,
    98,
    97,
    99,
    107,
    103,
    114,
    111,
    117,
    110,
    100,
    45,
    99,
    111,
    108,
    111,
    114,
    58,
    32,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    49,
    48,
    48,
    59,
    13,
    10,
    32,
    32,
    125,
    13,
    10,
    125,
    13,
    10,
  ],
  "type": "Buffer",
}
`;
